tmt/utils.py

```diff
@@ -662,6 +662,15 @@ def _workdir_init(self, id_: Optional[str] = None) -> None:
             self._workdir_cleanup(workdir)
 
         # Create the workdir
+        # Making sure WORKDIR_ROOT has 1777 permission if recreated
+        if workdir.startswith(WORKDIR_ROOT):
```

Comment:
I would blend these two conditions into one, and we would spare one indentation

Reply:
OKAY, autopep8 collapses it a bit but whatever.